### ISSUE TITLE
Stop Depending on SSLv2 symbols

### ISSUE THREAD

**Issue by tidux** [rg3/youtube-dl#8880]
SSLv2 is being completely removed from recent versions of OpenSSL and LibreSSL.  Blindly importing instead of importing only the symbols actually needed leads to errors like this on platforms without SSLv2:

$ youtube-dl -U
Traceback (most recent call last):
  File "/opt/local/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/opt/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/jon/bin/youtube-dl/__main__.py", line 16, in <module>
  File "/home/jon/bin/youtube-dl/youtube_dl/__init__.py", line 15, in <module>
  File "/home/jon/bin/youtube-dl/youtube_dl/options.py", line 7, in <module>
  File "/home/jon/bin/youtube-dl/youtube_dl/downloader/__init__.py", line 3, in <module>
  File "/home/jon/bin/youtube-dl/youtube_dl/downloader/common.py", line 8, in <module>
  File "/home/jon/bin/youtube-dl/youtube_dl/utils.py", line 27, in <module>
  File "/opt/local/lib/python2.7/ssl.py", line 97, in <module>
    import _ssl             # if we can't import it, let the error propagate
ImportError: ld.so.1: python2.7: fatal: relocation error: file /opt/local/lib/python2.7/lib-dynload/_ssl.so: symbol SSLv2_method: referenced symbol not found

This is on Illumos.

**Comment by yan12125** [comment]
Duplicate of #8767. It's not a fault of youtube-dl but Python. You'll need to rebuid your Python against new OpenSSL.
